### PR TITLE
Apply code checks to the RecoPixelVertexing packages

### PR DESCRIPTION
**Pull request by fwyzard**
Apply `clang-tidy` automated code changes.
Apply `clang-format` automated code formatting.

### CODE REVIEW

**Comment by fwyzard** [comment]
@cmsbuild, please test

File: RecoPixelVertexing/PixelLowPtUtilities/bin/ClusterShapeAnalyzer.cpp [hunk 1]
```diff
@@ -13,354 +13,343 @@
...
+  namespace po = boost::program_options;
+  using namespace cluster_shape_filter;
 
-        cluster_shape_filter::ClusterAnalyzer analyzer(args);
-        analyzer.analyze();
+  Arguments args;
+  po::options_description desc("Cluster shape filter analyzer");
+  std::ostringstream ss_loss;
+  ss_loss << std::setprecision(4) << args.loss;
+  desc.add_options()("help", "print help message")(
```
Comment (perrotta) [inline RecoPixelVertexing/PixelLowPtUtilities/bin/ClusterShapeAnalyzer.cpp:325]: The automatic reformatting here looks rather weird...

Comment (fwyzard) [reply RecoPixelVertexing/PixelLowPtUtilities/bin/ClusterShapeAnalyzer.cpp:325]: indeed... looks like `clang-format` doesn't like `boost::program_options` syntax much

**Comment by fwyzard** [comment]
@cmsbuild, please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/29895	b5dbb0f 
- technical

**Comment by silviodonato** [comment]
+1

**Pull request merged**